frontend/src/constants/screens.ts

```diff
@@ -12,28 +12,29 @@
 /**
  * mapping of breakpoint names to the lower-bound of their screen width range
  */
-const SCREEN_SIZES = Object.freeze(
+export const SCREEN_SIZES = Object.freeze(
```

Comment:
`/** @type {const} */ ()` in JSDoc corresponds to TypeScript's `as const`. So, we need to remove `/**@type {const} */()` wrapping this object, and add `as const` to the end. Make sure that you remove the wrapping `()` parentheses, too (not the parentheses from `Object.freeze` call, though).

This should be done to the other exports, too.